src/main/resources/ttc.properties

```diff
@@ -94,6 +93,9 @@ de.konfidas.ttc.messages.systemlogs.logoutCauseNotFound = Fehler beim Parsen des
 de.konfidas.ttc.messages.systemlogs.roleNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld role wurde nicht gefunden.
 de.konfidas.ttc.messages.systemlogs.timeAfterUpdateNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld timeAfterUpdate wurde nicht gefunden
 de.konfidas.ttc.messages.systemlogs.timeBeforeUpdateNotFoud = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil timeBeforeUpdate wurde nicht gefunden
```

Comment:
Pflichteil -> Pflichtfeld (also in the subequent ones)

Reply:
Ermmm, not done. Lines 94, 95 as well.

Reply:
done, code is outdated